File: Programming/ecobike_rental/lib/model/core/address.dart
class Address {
  String addressName;
  int longitude;
  int latitude;

  Address({this.addressName, this.longitude, this.latitude});

  Address.fromJson(Map<String, dynamic> json) {
    addressName = json['addressName'];
    longitude = json['longitude'];
    latitude = json['latitude'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['addressName'] = this.addressName;
    data['longitude'] = this.longitude;
    data['latitude'] = this.latitude;
    return data;
  }
}

File: Programming/ecobike_rental_server/TKXDPM_API/Model/Address.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace TKXDPM_API.Model
{
    public class Address
    {
        [Key] public int AddressId { get; set; }
        [Column(TypeName = "varchar(255)")] public string AddressName { get; set; }
        public float Longitude { get; set; }
        public float Latitude { get; set; }
    }

    public class AddressResponse
    {
        public string AddressName { get; set; }
        public float Longitude { get; set; }
        public float Latitude { get; set; }
    }
}